sections/home/ImpactSection.jsx

```diff
@@ -3,32 +3,59 @@ import styled from "styled-components";
 import ImpactGraphic1 from "../../public/home/impact-section-graphic-1.svg";
 import ImpactGraphic2 from "../../public/home/impact-section-graphic-2.svg";
 import ImpactGraphic3 from "../../public/home/impact-section-graphic-3.svg";
-import { ImpactCard } from "../../components";
+import { ImpactCard, Text, Button } from "../../components";
 import { media } from "../../utils";
+import { baseTheme } from "../../theme";
+import Link from "next/link";
```

Comment:
This should be at the top of the file. It always goes importing external packages first and then own files

Reply:
baseTheme as well? I saw baseTheme near the end of the WhatisHappening section and now it looks like this: 
![image](https://user-images.githubusercontent.com/67667948/126674253-db7f6267-5c4c-4b58-ab20-9da7e15a4872.png)
Is this good?

Reply:
yes that's fine. baseTheme is our own files so it does go at the end.